test-it/karma/events/ControlEvents.Test.js

```diff
@@ -119,4 +164,29 @@ describe('ValueChanged event tests', function () {
             done();
         });
     });
+
+    it('Verify event occurrence passes correct old and new value', function (done) {
```

Comment:
Please name this test as per BDD (See the first paragraph here: https://jasmine.github.io/tutorials/your_first_suite)

Reply:
I've updated all the names in the file to reflect the naming convention as best as I could